spyder/plugins/outlineexplorer/tests/test_widgets.py

```diff
@@ -17,50 +19,22 @@
 # Local imports
 from spyder.plugins.outlineexplorer.editor import OutlineExplorerProxyEditor
 from spyder.plugins.outlineexplorer.widgets import (
-    OutlineExplorerWidget, FileRootItem, FunctionItem, CommentItem,
-    CellItem, ClassItem)
+    OutlineExplorerWidget, FileRootItem, SymbolStatus, TreeItem)
 from spyder.plugins.editor.widgets.codeeditor import CodeEditor
 
+HERE = osp.abspath(osp.dirname(__file__))
+ASSETS = osp.join(HERE, 'assets')
+SUFFIX = 'test_widgets'
 
-TEXT = ("# -*- coding: utf-8 -*-\n"
-        "\n"
-        "# %% functions\n"
-        "\n"
-        "\n"
-        "# ---- func 1 and 2\n"
-        "\n"
-        "def func1():\n"
-        "    for i in range(3):\n"
-        "        print(i)\n"
-        "\n"
-        "\n"
-        "def func2():\n"
-        "    if True:\n"
-        "        pass\n"
-        "\n"
-        "\n"
-        "# ---- func 3\n"
-        "\n"
-        "def func3():\n"
-        "    pass\n"
-        "\n"
-        "\n"
-        "# %% classes\n"
-        "\n"
-        "class class1(object):\n"
-        "\n"
-        "    def __ini__(self):\n"
-        "        super(class1, self).__init__()\n"
-        "\n"
-        "    def method1(self):\n"
-        "        if False:\n"
-        "            pass\n"
-        "\n"
-        "    def method2(self):\n"
-        "        try:\n"
-        "            assert 2 == 3\n"
-        "        except AssertionError:\n"
-        "            pass")
+AVAILABLE_CASES = ['text']
+CASES = {
+    case: {
+        'file': osp.join(ASSETS, '{0}_{1}.py'.format(case, SUFFIX)),
+        'data': osp.join(ASSETS, '{0}_{1}.json'.format(case, SUFFIX)),
+        'tree': osp.join(ASSETS, '{0}_exp_{1}.json'.format(case, SUFFIX))
+    }
+    for case in AVAILABLE_CASES
+}
```

Comment:
Why is this necessary if there's only one available case (i.e. `text`)? What other kind of cases there could be?

Reply:
This allows to add more test cases on the future if necessary